docs/streaming-chat-completions-route.md

```
@@ -0,0 +1,217 @@
+     `data: {json}` events for `Ok` chunks.
+   - **On `Err`**: Yields `event: error` + `data: {message}`, then transitions
+     to `Done`.
+   - **On channel close (`None`)**: Yields `data: [DONE]`, then transitions to
+     `Done`.
+   - **`Done`**: The stream terminates (returns `None` from `unfold`).
+5. The stream is wrapped in `axum::response::sse::Sse` with
+   `KeepAlive::default()`.
+
+### State Machine Diagram
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)